dsm_novamid1070_175.xml.fdm_material

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+Generic ABS profile. The data in this file may not be correct for your specific machine.
+-->
+<fdmmaterial xmlns="http://www.ultimaker.com/material">
+    <metadata>
+        <name>
+            <brand>DSM</brand>
+            <material>Thermoplast</material>
```

Comment:
Thermoplast is an even broader name which includes basically all materials we can print with (specifically, all materials that become soft when heated). From the internet I gather that this material is a PA copolymer.